elements/bulbs-video/elements/summary.js

```diff
@@ -24,6 +24,19 @@ export function VideoSummaryView (props) {
     );
   }
 
+  let campaign;
+  if (video.tunic_campaign_url) {
+    campaign = (
+      <campaign-display
+        data-track-action={props.campaignTrackAction}
+        placement={props.campaignPlacement}
+        preamble-text={props.campaignPreamble}
+        src={video.tunic_campaign_url}
+        name-only
```

Comment:
I've tried that as well, in every case it only takes the first attribute defined on the element.